app/models/agents/post_agent.rb

```diff
@@ -162,22 +171,29 @@ def normalize_response_headers(headers)
       }
     end
 
-    def handle(data, payload = {}, headers)
-      url = interpolated(payload)[:post_url]
+    def handle(data, event = Event.new, headers)
```

Comment:
True, I guess the `headers` parameter was added as a third when the Agent got support for specifying the headers. [check](https://github.com/kreuzwerker/DKT.huginn/blob/c744ddf80c6b20a404e419c6afe06bcd5979b127/app/models/agents/post_agent.rb#L146) is calling `handle` with only two parameters.